seagence/README.md

```diff
@@ -0,0 +1,40 @@
+# Seagence for Datadog
+
+## Overview
+
+[Seagence][1] is a defect monitoring platform for Java applications that detects defects including root cause in real-time.
+
+### Description
+
+Tired of debugging unknown defects impacting your customer experience? Using revolutionary ExecutionPath technology, discover how Seagence detects unknown defects caused by various issues like swallowed exceptions, multithreading issues, and others including defects that disguise in a 200 HTTP response code in your production Java applications. Seagence is the only solution that can detect unknown defects as they manifest. You fix your code without debugging.
+
```

Comment:
We avoid marketing material in integrations-extras. Please remove since this is marketing copy

Reply:
Removed.